stl/inc/yvals_core.h

```diff
@@ -66,6 +66,7 @@
 // P1164R1 Making create_directory() Intuitive
 // P1165R1 Consistently Propagating Stateful Allocators In basic_string's operator+()
 // P1902R1 Missing Feature-Test Macros 2017-2019
+// P2251R1 Require span And basic_string_view To Be Trivially Copyable

```

Comment:
It feels weird to me to have this in the unconditional section when neither `span` nor `basic_string_view` is available. I think it might make more sense to duplicate it. Once in the `_HAS_CXX17` section suffixed with `[basic_string_view]`, and once in the `_HAS_CXX20` section suffixed with `[span]`.

This is novel for the STL, so let's get feedback from other contributors before deciding what to do here.

Reply:
I think that's a reasonable suggestion. The precedent that we can follow is:
https://github.com/microsoft/STL/blob/178b8406a5647507e759442960f17cd33045cdf5/stl/inc/yvals_core.h#L118-L119
https://github.com/microsoft/STL/blob/178b8406a5647507e759442960f17cd33045cdf5/stl/inc/yvals_core.h#L205-L206